sdk/agrifood/azure-agrifood-farming/README.md

```diff
@@ -1,4 +1,4 @@
-# Azure AgriFood Farming client library for Python
+# Azure FarmBeats client library for Python
```

Comment:
I feel like this should stay AgriFood Farming since that's the name of the package, but also it's your service, so I shall leave it to your discretion

Reply:
cc @lmazuel in case he's interested

Reply:
While AgriFood farming is the package name, the commercial product name is FarmBeats. We want SEO optimization geared towards the latter. Hence, prefer Farmbeats.